### ISSUE TITLE
Reading a row into a Decodable problem

### ISSUE THREAD

**Issue by a72** [yaslab/CSV.swift#69]
![image 2018-11-23 15 35 57](https://user-images.githubusercontent.com/12465993/48943822-82b4eb00-ef35-11e8-9a1a-54c715d0e32e.jpg)

**Comment by yaslab** [comment]
The master branch is still working, and the Decodable function still does not work well. Please wait a while until I tag the release.

**Comment by yaslab** [comment]
I officially released Decodable function as version 2.3.0. Since the decoding method has been changed to use the `CSVRowDecoder` class, please refer to the following code.

```swift
import CSV

struct DecodableExample: Decodable {
    let intKey: Int
    let stringKey: String
    let optionalStringKey: String?
}

let csv = """
    intKey,stringKey,optionalStringKey
    1234,abcd,
    5678,efgh,ijkf
    """

var records = [DecodableExample]()
do {
    let reader = try CSVReader(string: csv, hasHeaderRow: true)
    let decoder = CSVRowDecoder()
    // Go to the next row with the `CSVReader#next()` method.
    while reader.next() != nil {
        // Decode the current row into type `DecodableExample` 
        // with the `CSVRowDecoder#decode()` method.
        let row = try decoder.decode(DecodableExample.self, from: reader)
        records.append(row)
    }
} catch {
    // Invalid row format
    print("\(error)")
}
print(records)
```
